client/my-sites/sidebar/manage-menu.jsx

```diff
@@ -185,7 +184,7 @@ class ManageMenu extends PureComponent {
 
 		// Hide the sidebar link for multiple site view if it's not in calypso, or
 		// if it opts not to be shown.
-		const isEnabled = config.isEnabled( menuItem.config );
+		const isEnabled = ! menuItem.config || config.isEnabled( menuItem.config );
```

Comment:
Can this be true for other menu items/features as well?

Reply:
There are several additional feature flags in this file that are also obsolete: `manage/pages`, `manage/plugins`, ...

Until now, all menu items had an associated feature flag, although in most cases always true, and always had the `config` prop. After this patch, the `config` prop is optional. If it's present, a feature flag is checked, it it's not, the menu item is assumed to be always enabled.

I hope I answered the question 🙂